### PR TITLE
feat/table-styled

### PR DESCRIPTION
**Pull request by DaminiPandey**

### CODE REVIEW

**Review by Viraj-10 — APPROVED** [review]

**Comment by nickvallee** [comment]
@makkarMeenu hello! Anyway way we could get a second review on this some time soon? it would be nice to have access to the Table component in gluestack 😄 .

**Review by Rhuansantos — APPROVED** [review]

**Comment by Rhuansantos** [comment]
Hey @Viraj-10 any updates on what version this will be released? 

thank you!

**Comment by Viraj-10** [comment]
Table is released with v2 https://gluestack.io/ui/docs/components/table

**Pull request closed without merging**
